### PR TITLE
Rework form sugg

### PR DESCRIPTION
**Pull request by AlexisFLT**

### CODE REVIEW

**Review by Sax593 — DISMISSED** [review]

File: frontend/src/pages/Home/Home.jsx [hunk 1]
```diff
@@ -6,13 +6,16 @@ import "./Style.scss";
 
 export default function Home() {
   const [suggest, setSuggest] = useState([]);
+  const [, setPriority] = useState(null);
```
Comment (LoicBrassart) [inline frontend/src/pages/Home/Home.jsx]: What's this ? You update a state you don't keep

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
One incomprehensible Evil

File: frontend/src/pages/Home/Home.jsx (hunk 1 above)
Comment (AlexisFLT) [reply frontend/src/pages/Home/Home.jsx]: yes we know, it was temporary. I will fix it

**Review by Sax593 — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**